src/Events/Event.php

```diff
@@ -0,0 +1,208 @@
+    {
+        $this->output->write("");
+        $this->output->write("You encountered a merchant's ship in need of repairs. They ask of you to deliver their cargo and you will be rewarded with half of their payment.");
+        $options = ["Yes" => "Yes", "No" => "No"];
+        $result = $this->output->getCli()->radio("Do you agree to help them and deliver their cargo?", $options)->prompt();
+        $this->output->write("");
+        if ($result === "Yes") {
+            $this->output->write("");
+            $this->output->write("You agreed to deliver their cargo.");
+            $this->output->write("");
+            $questlog->addQuest(new Quest(cargo: $questlog->getRandomCargo(), destination: $hyperdrive->getRandomPlanet(), completed: false, main: false, exp: 2500, reward: $questlog->generateReward()));
+        }
+        if ($result === "No") {
+            $this->output->write("");
+            $this->output->write("The merchant wasn't happy with your reply but you decided it's not your problem that he has a broken ship. You decide to leave him alone.");
+            $this->output->write("");
+        }
+    }
+
+    private function asteroidBeltEvent(Pilot $player, Ship $playerShip): void
+    {
+        $this->output->write("");
+        $this->output->info("You encountered an Asteroid belt! You lost some fuel while trying to maneuver through them.");
+        $fuelLost = 5 * (10 - $player->getSkill());
+        $playerShip->loseFuel($fuelLost);
+    }
+
+    private function spaceCombatEvent(Pilot $player, Ship $playerShip, HyperdriveNavigator $hyperdrive, QuestLog $questlog, PlanetSurface $surface, Event $event): void
+    {
+        $this->output->write("");
+        $this->output->info("You've been attacked!");
+        $enemies = new Enemies($this->output);
+        $combat = new Combat($this->output);
+        $combat->landingOrFighting($player, $playerShip, $enemies->getEnemyShips(), $surface, $event, $questlog, $hyperdrive);
+
+    }
+
+
+    public function randomLandEvents(Pilot $player, Ship $playerShip): void
+    {
+        $random = rand(1, 10);
+
+        if ($random == 7) {
+            $this->robberyEvent($player);
+        }
+        if ($random == 8) {
+            $this->reputationEvents($player, $playerShip);
+        }
+        if ($random == 9) {
+            $this->gamblingEvent($player);
+        }
+        if ($random == 10) {
+            $this->racingEvent($player);
+        }
+    }
+
+    private function robberyEvent(Pilot $player): void
+    {
+        $creditsLost = rand(0, 200) + 1000;
+        $this->output->write("");
+        $this->output->write("After walking through the city for a while you realized some of your credits has been stolen!");
+        $this->output->info("You lost $creditsLost credits");
+        $player->payCredits($creditsLost);
+    }
+
+    private function gamblingEvent(Pilot $player): void
+    {
+        $this->output->write("");
```

Comment:
Jeśli tak często robisz puste linie, to może warto byłoby zrobić metodę, która by od tego była?